### ISSUE TITLE
Cities / Addresses Not Working

### ISSUE THREAD

**Issue by lxle** [archisman-panigrahi/typhoon#11]
Entering any City or Address does not work, whether its a numerical value or a New York, NY style address. Nothing works any longer, something in the yahoo api must have changed. Updated needed here and then also applied to the ppa for easy updating.

**Comment by Vort** [comment]
Here is the problem: "To increase the security measure and better serving our YDN customers, weather data source will enforce OAuth 1 starting March 15, 2016." (https://developer.yahoo.com/weather/)

**Comment by apandada1** [comment]
It seems they have closed this service. http://weather.yahooapis.com/forecastrss?w=2442047

**Comment by apandada1** [comment]
My existing installation is still working but cannot change the location

**Comment by lxle** [comment]
same here, vorts comment is probably the most valid

**Comment by lxle** [comment]
might be helpful https://developer.forecast.io/

**Comment by ylluminate** [comment]
Meteorologist has been going through the same issues:
https://sourceforge.net/p/heat-meteo/discussion/268087

Specifically: https://sourceforge.net/p/heat-meteo/discussion/268087/thread/1c667652/

Discussion of options:
https://sourceforge.net/p/heat-meteo/discussion/268087/thread/9cdf548b/

I have to lean towards forecast.io as well.  I've been finding it very useful as of late and the API is great.

**Comment by lxle** [comment]
are you abandoning this project now?

**Comment by apandada1** [comment]
@lxle I am sorry for the inconvenience. I just don't get enough time to debug.

**Comment by lxle** [comment]
fixing it or not?

**Comment by lxle** [comment]
Archisman Panigrahi wrote:
>
> @lxle <https://github.com/lxle> I am sorry for the inconvenience. I
> just don't get enough time to debug.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/apandada1/typhoon/issues/11#issuecomment-210302448>
>
is the project dead or isn't?

simple yes or no


if it is, i'm ripping it out of the lxle linux distribution

**Comment by mcris** [comment]
Yes, please rip this app from LXLE and if you can find another one (Light Linux has such weather app)

**Comment by archisman-panigrahi** [comment]
Fixed
